Documentation/Documentation.css

```diff
@@ -164,6 +166,8 @@ fieldset{
 .markup{
     background-color: var(--box-shadow);
     font-weight: 500;
+    padding: 0.3rem;
```

Comment:
for smaller values less than 1rem try using px

Reply:
okay done thanks for suggestion